tempest/src/main/kotlin/app/cash/tempest/internal/Schema.kt

```diff
@@ -263,6 +279,9 @@ internal data class ItemType(
       return secondaryIndexes.toMap()
     }
 
+    private val KProperty<*>.shouldIgnore: Boolean
+      get() = findAnnotation<Ignore>() != null || findAnnotation<Transient>() != null
```

Comment:
I think you’re better off with `Transient` only. Otherwise we might accidentally annotate tests with Tempest’s `@Ignore` and get confused

Reply:
Good idea! Fixed in a follow up PR: https://github.com/cashapp/tempest/pull/11